scripts/test_hard_gt.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+# Script for making "test_hard_gt.jsonl"
+## https://github.com/facebookresearch/mmbt/issues/6
+
+import argparse
+import copy
+import json
+import os
+
+def get_args(parser):
+    parser.add_argument("--task", type=str, default="mmimdb", choices=["mmimdb", "vsnli", "food101"])
+    parser.add_argument("--mmdb_path", type=str, default="/home/ubuntu/data/mmimdb/mmimdb")
```

Comment:
mmimdb_path ?

Reply:
I would suggest just having one single `args.path` argument that points to the dataset path.

Reply:
Will change as commented below